src/AgGrid.fs

```diff
@@ -4,18 +4,200 @@ module Feliz.AgGrid
 open System
 open Fable.Core
 open Fable.Core.JsInterop
-
 open Feliz
 
 let agGrid : obj = import "AgGridReact" "ag-grid-react"
 
-importAll "ag-grid-community/styles/ag-grid.css"
-importAll "ag-grid-community/styles/ag-theme-alpine.css"
-importAll "ag-grid-community/styles/ag-theme-balham.css"
-importAll "ag-grid-community/styles/ag-theme-material.css"
+// User should load the CSS files in their own project
+//importAll "ag-grid-community/styles/ag-grid.css"
+//importAll "ag-grid-community/styles/ag-theme-alpine.css"
+//importAll "ag-grid-community/styles/ag-theme-balham.css"
+//importAll "ag-grid-community/styles/ag-theme-material.css"
+
+// https://www.ag-grid.com/javascript-data-grid/row-object/

```

Comment:
I like having the link to the docs easily available. There are two changes I'd consider:

- Use doc comments, to help library consumers.
- Always link to the react-data-grid version (rather than the javascript-data-grid one).